pkgs/tools/audio/nanotts/default.nix

```
@@ -12,8 +12,8 @@ stdenv.mkDerivation {
   };
 
   strictDeps = true;
-  nativeBuildInputs = [ autoconf automake libtool ];
-  buildInputs = [ popt alsa-lib ];
+  nativeBuildInputs = [ autoconf automake libtool makeWrapper ];
+  buildInputs = [ popt alsa-lib alsa-plugins ];
```

It's not necessary to have this on buildInputs for the wrapper

```suggestion
  buildInputs = [ popt alsa-lib ];
```